### PR TITLE
Bruhspace event text is restored to mith's version

### PR DESCRIPTION
**Pull request by zzzmike**
## About The Pull Request

After moko said in discord that bruhspace anomaly was a funnier name, I decided to make this. Restores bruhspace event text to Mith's version at https://github.com/TheSwain/Fulpstation/pull/81 (thank you John for bringing this event back)

## Why It's Good For The Game

It's more familiar for the people who remember the original event.

## Changelog

:cl:
balance: bruhspace anomaly event text is restored to Mith's version
/:cl:

### CODE REVIEW

**Comment by Pepsilawn** [comment]
Bruh

**Comment by SmoSmoSmoSmok** [comment]
bruh

File: fulp_modules/features/events/bruhspace_anomaly.dm [hunk 1]
```diff
@@ -12,4 +12,4 @@
 		all_players.say("bruh")
 
 /datum/round_event/bruh_moment/announce(fake)
-	priority_announce("An incoming Bruh Moment has been detected. Please stand by.", "Anomaly Alert")
+	priority_announce("NanoTrasen is issuing a Bruh Moment warning. Please stand by.", "Anomaly Alert")
```
Comment (JohnFulpWillard) [inline fulp_modules/features/events/bruhspace_anomaly.dm:15]:
```suggestion
	priority_announce("Nanotrasen is issuing a Bruh Moment warning. Please stand by.", "Anomaly Alert")
```

**Comment by zzzmike** [comment]
woops I didn't check this until I got the stale email. anyways merged john's suggestions, thx john

**Comment by zzzmike** [comment]
(john willard code review gif here)

**Comment by SgtHunk** [comment]
bruh

**Pull request merged**